Makefile

```diff
@@ -73,6 +73,10 @@ DOCKER_FLAGS =
 DOCKER_BUILD ?= $(DOCKER) run --rm -v $(DIR):$(BUILDMNT) $(DOCKER_FLAGS) -w $(BUILDMNT) $(BUILD_IMAGE) /bin/sh -c
 GO_BUILD ?= CGO_ENABLED=0 $(GO_SYSTEM_FLAGS) go build -o $(BINARY) $(VERBOSE_FLAG) -ldflags="-s -w -X $(GOTARGET)/pkg/buildinfo.Version=$(GIT_VERSION) -X $(GOTARGET)/pkg/buildinfo.GitSHA=$(GIT_REF_LONG)" $(GOTARGET)
 
+# Kind images
+K8S_PATH ?= $(shell cd ~/go/src/github.com/kubernetes/kubernetes && pwd)
```

Comment:
Perhaps this could use `GOPATH` (it's defined above) for referencing the Kubernetes path?